### ISSUE TITLE
Authentication method '' is not supported

### ISSUE THREAD

**Issue by mguinness** [PomeloFoundation/Pomelo.EntityFrameworkCore.MySql#123]
Using MySQL version 5.1.38 on Windows I get the following error, but using MySQL version 5.6.27 on Linux I do not.  Also this isn't an issue with either the official MySql or Sapient Guardian providers,  Any ideas/suggestions?

```
System.NotSupportedException: Authentication method '' is not supported.
   at MySql.Data.Serialization.MySqlSession.<ConnectAsync>d__22.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at MySql.Data.MySqlClient.ConnectionPool.<GetSessionAsync>d__0.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at MySql.Data.MySqlClient.MySqlConnection.<CreateSessionAsync>d__55.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at MySql.Data.MySqlClient.MySqlConnection.<OpenAsync>d__11.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at MySql.Data.MySqlClient.MySqlConnection.Open()
   at Microsoft.EntityFrameworkCore.Storage.Internal.MySqlRelationalConnection.Open()
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable.Enumerator.BufferlessMoveNext(Boolean buffer)
   at Microsoft.EntityFrameworkCore.Query.QueryMethodProvider.<_ShapedQuery>d__3`1.MoveNext()
   at Microsoft.EntityFrameworkCore.Query.Internal.MySqlQueryingEnumerable`1.MySqlEnumerator.MoveNext()
   at System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   at lambda_method(Closure , QueryContext )
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryCompiler.<>c__DisplayClass20_0`1.<CompileQueryCore>b__0(QueryContext qc)
   at System.Linq.Queryable.FirstOrDefault[TSource](IQueryable`1 source, Expression`1 predicate)
   at Microsoft.EntityFrameworkCore.Internal.EntityFinder`1.Find(Object[] keyValues)
   at DataQuery.Controllers.HomeController.Index() in C:\Projects\DataQuery\src\DataQuery\Controllers\HomeController.cs:line 55
```

### Further technical details

MySQL version: 5.1.38-community
Operating system: Windows Server 2008 R2
Pomelo.EntityFrameworkCore.MySql version: 1.1.0-rtm-10003

**Comment by Kagamine** [comment]
Please move this issue to mysql-net/MySqlConnector.

**Comment by bgrainger** [comment]
@Kagamine You should be able to fix this by updating to [MySqlConnector 0.7.2](https://www.nuget.org/packages/MySqlConnector/0.7.2).

**Comment by Kagamine** [comment]
@bgrainger thx
